### ISSUE TITLE
Incorrect space usage showing on Swiss v0.5r1138

### ISSUE THREAD

**Issue by Spade013** [emukidid/swiss-gc#568]
I'm using the sd2sp2 with a 256GB micro sd card (specifically this one: https://www.bestbuy.com/site/sandisk-256gb-microsdxc-uhs-i-memory-card-for-nintendo-switch/6350363.p?skuId=6350363).
The memory on both the standard and carousel browser types are not showing correctly even though I have several games loaded using cleanrip.
Swiss is loaded using the SD Media Launcher method.
![Screenshot 2021-08-17 18-40-07](https://user-images.githubusercontent.com/35104774/129810753-a53fd418-af27-4742-bdaa-0bf5f630b80f.png)
![Screenshot 2021-08-17 18-41-14](https://user-images.githubusercontent.com/35104774/129810756-0535f9b7-c633-4232-907f-b7a02b9d41aa.png)

**Comment by Extrems** [comment]
Which operating system did you use to manipulate the SD card?

**Comment by Spade013** [comment]
Windows 10. I think I used Raspberry Pi Imager tool to format it to Fat32

**Comment by Extrems** [comment]
Have you tried chkdsk/fsck on the SD card?

**Comment by Spade013** [comment]
After multiple blue screens on my old pc and a download of a third party software, I finally got it working.
The software I used was the gui version of fat32format. http://ridgecrop.co.uk/index.htm?guiformat.htm
The built-in windows formatting using cmd was taking too long and my pc crashed 3 times while running it. The software not only worked, but it worked very fast.
![Screenshot 2021-08-18 01-21-58](https://user-images.githubusercontent.com/35104774/129841781-5df0ba01-d1ae-4a25-88e2-a1f8d93ba4e0.png)
